evo1/memory/base.py

```diff
@@ -56,8 +59,7 @@ def gli(self) -> int:
 
     @property
     def lvl(self) -> int:
```

Comment:
This is used to calculate Clink's attack/defense out of battle for the AttackPrediction calculations.